common/script_values/imperia_values.txt

```diff
@@ -832,139 +832,173 @@ imperia_qing_fragile_unity_value = {
 		}
 		add = 20
 	}
-	every_country = {
-		# Every one of these tags that are NOT a subject adds 10
-		limit = {
-			OR = {
-				AND = {
-					exists = c:XIN
-					this = c:XIN
-				}
-				AND = {
-					exists = c:MGL
-					this = c:MGL
-				}
-				AND = {
-					exists = c:MCH
-					this = c:MCH
-				}
-				AND = {
-					exists = c:XIB
-					this = c:XIB
-				}
-				AND = {
-					exists = c:TIB
-					this = c:TIB
-				}
-				AND = {
-					exists = c:SIH
-					this = c:SIH
-				}
-				AND = {
-					exists = c:HNA
-					this = c:HNA
-				}
-				AND = {
-					exists = c:YUE
-					this = c:YUE
-				}
-			}
-			NOT = {
-				is_subject_of = ROOT
-			}
-		}
-		add = 10
-	}
-	every_country = {
-		# Every one of these tags that are NOT a subject adds 5
-		limit = {
-			OR = {
-				this = c:FRS
-				this = c:KOR
-			}
-			NOT = {
-				is_subject_of = ROOT
-			}
-		}
-		add = 5
-	}
-	# Each of these countries add 10 if they dont exists
+	# If Qing doesn't have a presence in the following subject capitals...
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Xinjiang (Tarim Basin)
 			NOT = {
-				exists = c:XIN
+				any_state = {
+					state_region = s:STATE_TIANSHAN
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
-		add = 10
+		add = 5
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Mongolia (Urga)
 			NOT = {
-				exists = c:MGL
+				any_state = {
+					state_region = s:STATE_URGA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
-		add = 10
+		add = 5
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Eastern provinces (Southern Manchuria)
 			NOT = {
-				exists = c:MCH
+				any_state = {
+					state_region = s:STATE_SOUTHERN_MANCHURIA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 10
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Gansu (Xibei, Ningxia)
 			NOT = {
-				exists = c:XIB
+				any_state = {
+					state_region = s:STATE_NINGXIA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 10
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Tibet (Lhasa)
 			NOT = {
-				exists = c:TIB
+				any_state = {
+					state_region = s:STATE_LHASA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
-		add = 10
+		add = 5
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Sichuan
 			NOT = {
-				exists = c:SIH
+				any_state = {
+					state_region = s:STATE_SICHUAN
+					owner = {
+						OR = {
+							this ?= c:CHI
```

Comment:
ehm.. it is my understanding that ?= does an exist test on the left side, so it checks whether "this" exists and then checks whether it is c:CHI. So if c:CHI doesn't exist it would still throw errors according to my understanding. You just need to check exists = c:CHI the normal way I think.